Apps/W1/ContosoCoffeeDemoDataset/app/Service/Libraries/SvcDemoDataSetup.Table.al

```diff
@@ -0,0 +1,170 @@
+table 4765 "Svc Demo Data Setup"
+{
+    DataClassification = CustomerContent;
+
+    fields
+    {
+        field(1; "Primary Key"; Integer)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Primary Key';
+        }
+        field(2; "Is DemoData Populated"; Boolean)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Is DemoData Populated';
+        }
+        field(8; "Starting Year"; Integer)
+        {
+            DataClassification = CustomerContent;
```

Comment:
you can leave those blank, the table level classification will take over

Reply:
The danger of unexamined copy/paste from Mfg.  Fixing...